tests/pr_test.go

```diff
@@ -170,8 +170,9 @@ func TestAgentsSolutionInSchematics(t *testing.T) {
 	} else {
 
 		options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
-			Testing: t,
-			Prefix:  "obs-agents",
+			TerraformVersion: "terraform_v1.9.0",
```

Comment:
perhaps add a code comment above this which says:

`# explicitly setting version here until https://github.ibm.com/GoldenEye/issues/issues/11051 is addressed`